core/src/main/java/feast/core/config/FeastProperties.java

```diff
@@ -114,6 +115,30 @@ LoggingProperties loggingProperties() {
 
       /* Labels to identify jobs managed by this job coordinator */
       private Map<String, String> jobSelector = new HashMap<>();
+
+      /* Selectors to define featureSets that are responsibility of current JobManager */
+      private List<Selector> featureSetSelector = new ArrayList<>();
+
+      /* Specify names of stores that must not be used by current JobManager */
+      private List<String> blacklistedStores = new ArrayList<>();
+
+      /**
+       * Similarly to Store's subscription this selector defines set of FeatureSets. All FeatureSets
+       * that match both project and name will be selected. Project and name may use *
+       */
+      @Getter
+      @Setter
+      public static class Selector {
```

Comment:
We should maybe make this `FeatureSelector` to differentiate it from other selectors.